build/Build.cs

```diff
@@ -288,6 +294,7 @@ ModifiableFileWithRestoreContentsOnDispose ModifyTemplatedVersionAndProductFiles
     ModifiableFileWithRestoreContentsOnDispose UpdateMsiProductVersion()
     {
         var productWxsFilePath = RootDirectory / "installer" / "Octopus.Tentacle.Installer" / "Product.wxs";
+        var productWxsFile = new ModifiableFileWithRestoreContentsOnDispose(productWxsFilePath);
```

Comment:
ℹ️ We made a modification to this file before its creation. Now, we made sure it is created so we can capture the modification.